scripts/link-dependencies.sh

```diff
@@ -34,35 +34,11 @@ yarn unlink
 yarn link
 cd -
 
-if [ -d "../budibase-pro" ]; then
-  cd ../budibase-pro
-  echo "Bootstrapping budibase-pro"
-  yarn bootstrap
-
-  cd packages/pro
-  echo "Linking pro"
-  yarn unlink
-  yarn link
-
-  echo "Linking backend-core to pro"
-  yarn link '@budibase/backend-core'
-
-  echo "Linking types to pro"
-  yarn link '@budibase/types'
-
-  echo "Linking string-templates to pro"
-  yarn link '@budibase/string-templates'
-
-  cd ../../../budibase
-
-  echo "Linking pro to worker"
-  cd packages/worker && yarn link '@budibase/pro'
-  cd -
-
-  echo "Linking pro to server"
-  cd packages/server && yarn link '@budibase/pro'
-  cd -
-fi
+echo "Linking pro"
```

Comment:
Do we need to do this at all anymore? I'd have thought with it now being a package it should be auto symlinked like the other packages.

Aside, if that's not possible for some reason we should keep the `if [ -d "../budibase-pro" ]; then` check (updated for the correct path) so that open source contributors don't run into trouble with this script.

Reply:
This will be needed if `account-portal` needs pro

Reply:
Sure 👍